### PR TITLE
Instructions on `readme`

### PR DESCRIPTION
**Pull request by seongsujeong**
This PR adds installation and usage instruction for RTC.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,54 @@
...
+
+### Usage
+
+The command below generates the RTC product:
+
+```bash
+rtc_s1.py <path to rtc yaml file>
+```
+
+To compare the products (`.tif` or `.hd5`), use `rtc_compare.py`.
```
Comment (gshiroma) [inline README.md:46]:
```suggestion
To compare the RTC-S1 products, use `rtc_compare.py`.
```

File: README.md [hunk 2]
```diff
@@ -1,6 +1,54 @@
...
+
+```bash
+rtc_s1.py <1st file> <2nd file>
```
Comment (gshiroma) [inline README.md:49]:
```suggestion
python rtc_s1.py <1st product HDF5> <2nd product HDF5>
```

File: README.md [hunk 3]
```diff
@@ -1,6 +1,54 @@
 # RTC
 OPERA Radiometric Terrain-Corrected (RTC) Product
 
+
+🚨 This toolbox is undergoing **rapid development**. 🚨
+
+### Install
+
+The following will install COMPASS into a conda environment.
```
Comment (gshiroma) [inline README.md:9]:
```suggestion
Instructions to install RTC under a conda environment.
```

File: README.md [hunk 4]
```diff
@@ -1,6 +1,54 @@
...
+
+1. Download source code:
```
Comment (gshiroma) [inline README.md:11]:
```suggestion
1. Download the source code:
```

**Review by gshiroma — COMMENTED** [review]
Thank you for working on that @seongsujeong . Ideally, we'll add other ways to install RTC without `pip`, but we can do that later.

**Comment by seongsujeong** [comment]
@gshiroma Thanks for the quick turnaround. I've accepted all of your suggestions.

**Review by gshiroma — APPROVED** [review]
LGTM. Thank you, @seongsujeong

**Pull request merged**